components/VersionSection.tsx

```diff
@@ -56,61 +104,105 @@ export default function VersionSection({
-              <XCircle className="h-8 w-8 fill-destructive text-destructive-foreground" />
-              <Paragraph
-                className="text-destructive"
-                variant="smallText"
-                margin="none"
-              >
-                There was an error checking for updates.
-              </Paragraph>
-            </div>
-          )}
-        </div>
-      }
+      controlArea={!data?.semVerUpdateType ? statusContent : undefined}
     >
       <Paragraph>
         You are currently running Fresco v.{env.APP_VERSION} ({env.COMMIT_HASH}
         ).
       </Paragraph>
       <Paragraph>Your unique installation ID is: {installationID}</Paragraph>
+
+      {data?.semVerUpdateType && (
+        <>
+          <Alert
+            variant={data.semVerUpdateType === 'major' ? 'default' : 'info'}
+            className="mt-4"
+          >
+            <AlertTitle>
+              {' '}
+              <div className="mt-4 flex flex-row items-center space-x-2">
+                <Info className="h-8 w-8 text-info" />
+                <Paragraph
+                  className="text-info"
+                  variant="smallText"
+                  margin="none"
+                >
+                  A new version of Fresco is available: {data.newVersion}
+                </Paragraph>
+              </div>
+            </AlertTitle>
+            <AlertDescription>
+              <Paragraph>
+                To upgrade your Fresco version, you will need to sync your fork
+                with the latest version of the Fresco repository. For more
+                information, please refer to the{' '}
+                <Link href="https://documentation.networkcanvas.com/en/fresco/deployment/upgrading">
+                  upgrade documentation.
+                </Link>
+              </Paragraph>
+
+              <Paragraph>
+                {data.semVerUpdateType && (
+                  <Alert
+                    variant={
+                      data.semVerUpdateType === 'major'
+                        ? 'destructive'
+                        : 'default'
+                    }
+                  >
+                    <AlertTitle>
+                      {updateTypeMessages[data.semVerUpdateType].label}
+                    </AlertTitle>
+                    <AlertDescription>
+                      {updateTypeMessages[data.semVerUpdateType].message}
+                    </AlertDescription>
+                  </Alert>
+                )}
+              </Paragraph>
+
+              <Paragraph>
+                <article className="prose prose-a:text-blue-600 prose-headings:text-sm prose-headings:text-foreground prose-headings:font-extrabold prose-headings:foreground prose-headings:tracking-widest prose-headings:uppercase text-sm text-foreground">
+                  <Markdown>{data.releaseNotes}</Markdown>
+                </article>
+              </Paragraph>
+            </AlertDescription>
```

Comment:
Also issues here with the structure:

- AlertDescription renders a div with the paragraph variants. There's no need for nested paragraphs inside. If you need nested paragraphs - don't use the AlertDescription.
- As per the radix docs, the description is important for screen readers, so I would keep it to just the basic info (first paragraph). However, that paragraph doesn't actually directly say that there's a new version available!
- The nested alert should _not_ be wrapped inside a Paragraph! It is a block level element that renders a div, and divs inside of p is invalid HTML. A DIV is conceptually meant to represent a container, which has its own spacing and alignment relative to other elements. 
- You also conditionally render the Alert, but not the paragraph. This will lead to empty markup, which would be bad for screen readers, and also potentially will add additional spacing due to margin/padding.
- The same goes for putting an `<article>` inside of a paragraph - never do this. Article is a more semantic form of div, and is block level. 
- You used an invalid color there (blue-600).